walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -27,13 +28,28 @@
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        List<TimeRange> optionalAttendeesConsideredRanges = new ArrayList<>();
+
+        /* try to remove the timeRanges optional attendees can't make it to. If there's nothing left, ignore the optional attendees*/
+        optionalAttendeesConsideredRanges = availableRanges(events, request, true);
```

Comment:
Combine variable declaration and initialization unless you need to separate them: `List<TR> oACR = availableRanges(events, request, true);`

Additionally, for primitive type parameters (String, int, boolean, etc), add a comment to help readers quickly understand what it's for. Within Google the convention is `availableRanges(events, requests, /* considerOptionalAttendees= */ true);`

Reply:
noted and implemented!